axiom/nr_app_harvest.c

```diff
@@ -12,6 +12,25 @@
 #include "nr_app_harvest_private.h"
 #include "util_logging.h"
 
+/*
+ * The build pipeline for PHP 8.2 uses a Debian image that is built on a
+ * version of glibc that contains an newer glibc version symbol for `pow()`
+ * which is incompatible with older distributions, causing RPM installs to fail.
+ *
+ * To pin the version of `pow` to a version compatible with all NR PHP Agent
+ * supported OSes, we utilize `asm()` and `.symver` to instruct the linker to
+ * select an older version- in this case, 2.17.
```

Comment:
Symbol version is dependent on arch. Adjust the comment to be more generic and let the code do the specifics:
```suggestion
 * select an older version of the symbol.
```